app/src/main/java/org/wikipedia/createaccount/CreateAccountActivity.kt

```diff
@@ -126,49 +126,53 @@ class CreateAccountActivity : BaseActivity() {
 
     private val createAccountInfo: Unit
         get() {
-            disposables.add(ServiceFactory.get(wiki).authManagerInfo
-                    .subscribeOn(Schedulers.io())
-                    .observeOn(AndroidSchedulers.mainThread())
-                    .subscribe({ response ->
-                        val token = response.query?.createAccountToken()
-                        val captchaId = response.query?.captchaId()
-                        if (token.isNullOrEmpty()) {
-                            handleAccountCreationError(getString(R.string.create_account_generic_error))
-                        } else if (!captchaId.isNullOrEmpty()) {
-                            captchaHandler.handleCaptcha(token, CaptchaResult(captchaId))
-                        } else {
-                            doCreateAccount(token)
-                        }
-                    }) { caught ->
-                        showError(caught)
-                        L.e(caught)
-                    })
+            lifecycleScope.launch {
+                try {
```

Comment:
Would it be possible to use `CoroutineExceptionHandler` instead of a `try catch`?

Reply:
Applied in cde801e500c1b1ce72193e19273fc749cc299b62